main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+func check(err error) {
+	if err != nil {
+		panic(err)
```

Comment:
Your program should not `panic` where it's possible to exit the program using `log.Fatal`.